Components/Terrain/src/OgreTerrainLodManager.cpp

```diff
@@ -66,7 +66,11 @@ namespace Ogre
 
     bool TerrainLodManager::isOpen() const
     {
+#ifndef OGRE_HAS_CXX11_EXPLICIT_CONVERSION_OPERATORS
+        return mDataStream != 0;
```

Comment:
sorry, this should have been ´mDataStream.get() != 0;` as SharedPtr lacks a != operator for ptr. Also you can drop the ifdefs as this method will also work with OGRE_HAS_CXX11_EXPLICIT_CONVERSION_OPERATORS

Reply:
Ok, I'll change it. I wonder why it compiled without warnings.

Reply:
Should I drop the new `OGRE_HAS_CXX11_EXPLICIT_CONVERSION_OPERATORS` define as well since it's no longer needed?

Reply:
yes